### ISSUE TITLE
Issue with react router

### ISSUE THREAD

**Issue by jinxac** [owais/django-webpack-loader#61]
Somehow I cant get this working with react router. 

So following is my routing code:-

```
import { Router, Route, browserHistory, IndexRoute } from 'react-router';
import React from 'react';
import ReactDOM from 'react-dom';
import App from './App';
import About from './About';
import Contact from './Contact';
import Home from './Home';


ReactDOM.render((
     <Router history={browserHistory}>
      <Route path="/" component={App}>
       <IndexRoute component={Home}/>
       <Route path="/about" component={About}/>
       <Route path="/contact" component={Contact}/>
  </Route>
  </Router>
), document.getElementById('app'));
```

Following is the App component:-

```
class App extends React.Component{
  render(){
  var href = this.props.history.createHref('/about');
  return(
        <div>
                <div>hello router</div>
                <Link to='/about'>About</Link>
                 <Link to="/contact">Contact</Link>
                {this.props.children}
        </div>
  )
}
}
```

So the issue I am facing is when i click on about the link changes but the content does not. However now if i press browser back I reach home and then i press browser forward I reach about again and this time both url and content change.

So to check this out i used the same generated bundled js file in a static html and tried to access to that file after npm start, it worked fine over there. However when I used the same file in django using web pack loader here I see the issue mentioned above

**Comment by jinxac** [comment]
So after debugging a bit further and deep I see that when I use following in my html, the issue is seen:-

If i include javascripts.html in my index.html, the above issue is seen:-

`<!doctype html>
<html>

     <head>
         {% load staticfiles %}
         {% load pipeline %}
         {% stylesheet 'wl-font-awesome' %}
         <link href="{% static 'lib/bootstrap/dist/css/bootstrap.min.css' %}" rel="stylesheet">
         {% stylesheet 'wlmain' %}
         <meta charset=utf-8/>
         <title>My First React Router App</title>
     </head>
     <body>
     <div id="app"></div>
                {% load render_bundle from webpack_loader %}

  {% render_bundle 'main' %}
                {% load render_bundle from webpack_loader %}
     {%include 'javascript.html' %} 

     </body>
</html>`


Javascripts.html:-

`  {% load staticfiles %}
<script type="text/javascript" src="{% static 'js/check.js' %}"></script>
<script type="text/javascript" src="{% static 'lib/jquery/dist/jquery.min.js' %}"></script>
<script type="text/javascript" src="{% static 'lib/bootstrap/dist/js/bootstrap.min.js' %}"></script>
<script type="text/javascript" src="{% static 'js/home.js' %}"></script>`

So basically when i call  {% load staticfiles %}, the issue is seen. Can you please share your comments on this?

**Comment by jinxac** [comment]
So guys I did something really silly, I had a js file which was calling html.pushState :/ because of which URLS were changing :/ . I will go ahead and close the issue now. Sorry for the trouble.
